easybuild/easyconfigs/h/Hadoop/Hadoop-2.5.0-cdh5.3.1-native.eb

```diff
@@ -26,7 +26,7 @@ dependencies = [(java, javaver)]
 
 build_native_libs = True
 
-extra_native_libs = [('snappy', 'lib/libsnappy.so')]
+extra_native_libs = [('snappy', 'lib/libsnappy.so*')]
```

Comment:
I think it's best here in case any of the supported native libs require particular patterns. It's nicer to not need to update the block.